config/locales/en/coronavirus_landing_page.yml

```diff
@@ -0,0 +1,21 @@
+en:
+  coronavirus_landing_page:
+    video_section_heading: ""
+    video_section_link: "https://www.youtube.com/watch?v=TmrYuT0_fZA&rel=0"
+    video_section_transcription_summary: View the video transcript
+    video_section_transcription_text: |
+      <p>As winter approaches, we’ll be spending more time indoors. This will increase the risk of coronavirus spreading.</p>
```

Comment:
Including the markup here is probably the quickest way of doing this but it feels wrong to have markup in a translation file like this. Maybe at some point this could be a list of things and the code could iterate over them and insert the <p> tags inside the template? (non-blocking)

Reply:
Yeah, looks really nasty – copied the approach from transition. I would've used govspeak, but needs altering the video template and the transition page. will try and refactor this at a later point.